Gems/ROS2/Code/Source/RobotImporter/URDF/JointsMaker.cpp

```diff
@@ -89,20 +88,25 @@ namespace ROS2
                 followColliderEntity->Deactivate();
             }
             break;
-        case urdf::Joint::REVOLUTE:
+        case sdf::JointType::REVOLUTE:
             { // Hinge
                 jointComponent = followColliderEntity->CreateComponent<PhysX::EditorHingeJointComponent>();
                 followColliderEntity->Activate();
 
-                const double limitUpper = AZ::RadToDeg(joint->limits->upper);
-                const double limitLower = AZ::RadToDeg(joint->limits->lower);
+                using LimitType = decltype(joint->Axis()->Upper());
+                const double limitUpper = joint->Axis()->Upper() != AZStd::numeric_limits<LimitType>::infinity()
+                    ? AZ::RadToDeg(joint->Axis()->Upper())
+                    : AZ::RadToDeg(AZ::Constants::TwoPi);
+                const double limitLower = joint->Axis()->Lower() != -AZStd::numeric_limits<LimitType>::infinity()
+                    ? AZ::RadToDeg(joint->Axis()->Lower())
+                    : -AZ::RadToDeg(AZ::Constants::TwoPi);
```

Comment:
We have to clampe the value between (-2pi, 2pi) because the PhysX Revolute joint is [restricted](https://gameworksdocs.nvidia.com/PhysX/4.1/documentation/physxapi/files/classPxRevoluteJoint.html#ad28e13ae7149b0a09f8b5a9d8beaae60) to that those angles.